### PR TITLE
Update FindCommand Message Usage

### PR DESCRIPTION
**Pull request by jevan13**
## Describe your changes
- Separated long message usage into parts for readability and standardisation

## Issue ticket number and link
builds on #115 

## Checklist before requesting a review
- [X] I have performed a self-review of my code.
- [X] If it is a core feature, I have added thorough tests.

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/FindCommand.java [hunk 1]
```diff
@@ -16,16 +16,17 @@
...
     public static final String ARGUMENT_WORD = "all";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all employees and potential hires whose names "
-            + "contain any of the specified keywords (case-insensitive) and displays them as a list with "
-            + "index numbers.\n"
-            + "Parameters: (all/e/ph) KEYWORD [MORE_KEYWORDS]...\n"
-            + "Example: " + COMMAND_WORD + " " + ARGUMENT_WORD + " alice bob charlie";
+    public static final String COMMAND_WORD = "find";
+    public static final String MESSAGE_PURPOSE = "Finds all employees and/or potential hires whose names "
+            + "contain any of the specified keywords (case-insensitive) and displays them as a list with";
```
Comment (LimErnest) [inline src/main/java/seedu/address/logic/commands/FindCommand.java]: Did you add an additional with?

Comment (jevan13) [reply src/main/java/seedu/address/logic/commands/FindCommand.java]: Will fix the typo. Thank you!

Comment (InfinityTwo) [reply src/main/java/seedu/address/logic/commands/FindCommand.java]: Did you forget a full stop at the end as well?

Comment (jevan13) [reply src/main/java/seedu/address/logic/commands/FindCommand.java]: Will fix the typo. Thank you!

**Review by LimErnest — APPROVED** [review]
Wonderful!

**Pull request merged**